monero-outreach-docs/translations/ru/storyboard/wallet_quickstart_ru.md

```diff
@@ -0,0 +1,57 @@
+
+Лёгкие кошельки бывают двух типов: с защитой ключа просмотра и с возможностью поделиться ключом просмотра. Кошельки с защитой ключа просмотра сохраняют его полную конфиденциальность. Как следствие, при использовании таких кошельков приходится загружать блокчейн (или, по крайней мере, его часть) уже после создания кошелька, чтобы высчитывать баланс Monero на нём. Лёгкие кошельки с возможностью совместного использования ключа просмотра, с другой стороны, делятся ключом просмотра с сервером, на котором работает удалённый узел, чтобы вычислить баланс входящих средств на сервере. Такие эффективные кошельки иногда называют приложениями. Возможность совместного использования ключа просмотра снижает уровень анонимности, если не доверяете серверу, на котором работает узел, которому известен ваш ключ просмотра, поскольку имеется вероятность просмотра входящих транзакций. Тем не менее, следует отметить, что иногда владельцы кошельков делятся своим ключом просмотра в целях проведения аудита. Но это никак не позволяет перевести Monero куда-либо.
+
+Для обеспечения максимального уровня безопасности (которая обеспечивается даже в том случае, если ваше устройство заражено вредоносным программным обеспечением) можно использовать специализированные аппаратные кошельки. Аппаратные кошельки являются небольшими интерактивными электронными устройствами, которые хранят ваши приватные ключи и связываются с программным обеспечением основного устройства, используя только публичную информацию. Аппаратные кошельки гарантируют повышенную безопасность и обычно стоят $50-150.
+
+Учитывая всё это, следует рассмотреть следующие варианты: если вы хотите запустить собственный полный узел, то хорошим выбором станет официальный GUI или CLI-кошелёк Monero. Первый кошелёк поддерживает графический интерфейс пользователя, а второй имеет текстовый интерфейс. Оба кошелька можно конфигурировать так, что они будут поддерживать либо собственный полный узел, либо удалённый узел с повышением уровня анонимности. Эти кошельки можно загрузить по ссылке: [getmonero.org/downloads](https://getmonero.org/downloads/). С устройствами, работающими под операционной системой Android или iOS, рекомендуется использовать лёгкие кошельки с защитой ключа просмотра: Monerujo для Android (можно скачать на [monerujo.io](https://www.monerujo.io/) и Cake Wallet для iOS (можно скачать на [cakewallet.io](https://cakewallet.io/)). Если же вы хотите создать кошелёк с возможностью совместного использования ключа просмотра, то вам рекомендуется установить браузерный кошелёк MyMonero (доступен для скачивания на [mymonero.com](https://mymonero.com/)). Edge Wallet (который можно скачать на [edge.app](https://edge.app/)) является другим солидным лёгким кошельком с возможностью совместного использования ключа просмотра, доступным как для Android, так и для iOS.
```

Comment:
Will just post the missing links :):

https://web.getmonero.org/ru/downloads/

Reply:
fix